nomad_darwin.go

```diff
@@ -31,12 +31,19 @@ func init() {
 }
 
 func readDefaultForNoMAD(key string) (string, error) {
-	cmd := execCommand("defaults", "read", "com.trusourcelabs.NoMAD", key)
-	out, err := cmd.Output()
-	if err != nil {
-		return "", err
+	// check for key in managed preferences first
+	cmdManaged := execCommand("defaults", "read", "/Library/Managed Preferences/com.trusourcelabs.NoMAD.plist", key)
+	outManaged, errManaged := cmdManaged.Output()
+	if errManaged != nil {
+		// now check user preferences
+		cmdUser := execCommand("defaults", "read", "com.trusourcelabs.NoMAD", key)
+		outUser, errUser := cmdUser.Output()
+		if errUser != nil {
+			return "", errUser
+		}
+		return strings.TrimSpace(string(outUser)), nil		
```

Comment:
Ideally the "happy" return is at the end of the function (as per https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88 - "Put the happy return statement as the very last line"). You could make this happen by using the same name for `out` and not redefining it on the inner loop: `out, err := cmdUser.Output()` - then you could just fall out of the `if` to the return at the end and it will return the right thing.